### PR TITLE
CASMCMS-7620: update image refs for CASM-2670

### PR DESCRIPTION
**Pull request by rkleinman-hpe**
## Summary and Scope

Update and lint charts per requirements in CASM-2670.

## Issues and Related PRs

* Resolves CASMCMS-7620

### Tested on:

  * `mug`

### Test description:

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? N/A
- Were continuous integration tests run? If not, why? N/A
- Was upgrade tested? If not, why? Yes
- Was downgrade tested? If not, why? Yes
- Were new tests (or test issues/Jiras) created for this change? N/A

Testing manifest:

```yaml
apiVersion: manifests/v1beta1
metadata:
  name: sysmgmt-rkleinman
spec:
  charts:
  - name: cray-tftp
    namespace: services
    source: csm-algol60
    values:
      cray-service:
        containers:
          cray-tftp:
            image:
              repository: artifactory.algol60.net/csm-docker/unstable/cray-tftpd
      hmn_tftp_service_ip: 10.94.100.60
      nmn_tftp_service_ip: 10.92.100.60
      storage_shared_class_cray_tftp: ceph-cephfs-external
      storage_shared_size_cray_tftp: 5Gi
    version: 0.7620.129-20211122220431+a6f1fd6
  - name: cray-tftp-pvc
    namespace: services
    source: csm-algol60
    version: 0.7620.129-20211122220431+a6f1fd6
  sources:
    charts:
    - location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
      name: csm-algol60
      type: repo
```

## Risks and Mitigations

None

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by rbak-hpe — APPROVED** [review]

**Review by rbak-hpe — APPROVED** [review]

**Review by ecozzi-hpe — APPROVED** [review]

File: .github/workflows/charts-lint-test-scan.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Lint, test, and scan Helm charts
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```
Comment (jsollom-hpe) [inline .github/workflows/charts-lint-test-scan.yml:9]: Wish someone could tell me why workflow_dispatch is there but empty. I know it's a boilerplate form, but I'd still like to know.

File: kubernetes/cray-tftp/templates/nf_nat_tftp.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
           requests:
             cpu: ".1"
             memory: "20M"
-        image: "{{ $baseChartValues.imagesHost }}/baseos/alpine:3.12"
+        image: "artifactory.algol60.net/docker.io/library/alpine:3"
```
Comment (jsollom-hpe) [inline kubernetes/cray-tftp/templates/nf_nat_tftp.yaml:44]: I like letting everything but the major version float.

**Review by jsollom-hpe — APPROVED** [review]

File: .github/workflows/charts-lint-test-scan.yml (hunk 1 above)
Comment (rkleinman-hpe) [reply .github/workflows/charts-lint-test-scan.yml:9]: When it is empty that means you can kick off a github action, and there are no inputs that are necessary to kick off the action.

Comment (rkleinman-hpe) [reply .github/workflows/charts-lint-test-scan.yml:9]: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

File: kubernetes/cray-tftp/templates/nf_nat_tftp.yaml (hunk 2 above)
Comment (rkleinman-hpe) [reply kubernetes/cray-tftp/templates/nf_nat_tftp.yaml:44]: Agree, this is doing a very simple action with basic tools, so we shouldn't need to worry about minor releases.

**Pull request merged**